src/pybind/mgr/orchestrator/module.py

```diff
@@ -1135,8 +1135,8 @@ def _service_rm(self, service_name, force=False):
     @_cli_write_command(
         'orch apply',
         'name=service_type,type=CephChoices,strings=mon|mgr|rbd-mirror|crash|alertmanager|grafana|node-exporter|prometheus,req=false '
-        'name=dry_run,type=CephBool,req=false '
         'name=placement,type=CephString,req=false '
+        'name=dry_run,type=CephBool,req=false '
         'name=format,type=CephChoices,strings=plain|json|json-pretty|yaml,req=false '
         'name=unmanaged,type=CephBool,req=false',
         'Update the size or placement for a service or apply a large yaml spec')
```

Comment:
minor nit: the order of  params in the `_cli_write_command` decorator doesn't match the order seen in the actual func signature ...  e.g. `unmanaged` after `placement` or `format`?

maybe we should also clean these up to increase readability?